packages/beacon-state-transition/.mocharc.yaml

```diff
@@ -1,6 +1,8 @@
 colors: true
-require:
-  - ts-node/register
-  - ./test/setup.ts
```

Comment:
If the test setup is disabled, how is bls setup now? With the automatic top level await?

Reply:
Yeah